app/controllers/posts_controller.rb

```diff
@@ -1,6 +1,6 @@
 class PostsController < ApplicationController
   def index
-    @user = User.find(params[:user_id])
+    @user = User.includes(posts: { comments: :user }).find(params[:user_id])
     @posts = @user.posts
   end
```

Comment:
- [ ] One of the project's requirements for this milestone is to include the screenshots of logs in your terminal when your code had **_N+1 queries_** and screenshots of the logs after the **_N+1 queries_** were fixed. 😎 👍 

![image](https://github.com/zamcham/ThoughtTrail/assets/77106962/a17d26fc-9036-43e8-9ef1-fb9a2f66ac77)